service/stop.go

```diff
@@ -13,39 +14,27 @@ func (s *Service) Stop(c container.Container) error {
 		return err
 	}
 
-	if err := s.StopDependencies(c); err != nil {
-		return err
-	}
-	return c.DeleteNetwork(s.namespace())
-}
-
-// StopDependencies stops all dependencies.
-func (s *Service) StopDependencies(c container.Container) error {
-	var mutex sync.Mutex
-	var wg sync.WaitGroup
-	var err error
-	stop := func(d *Dependency) {
-		defer wg.Done()
-		errStop := d.Stop(c, s)
-		mutex.Lock()
-		defer mutex.Unlock()
-		if errStop != nil && err == nil {
-			err = errStop
+	var (
+		wg   sync.WaitGroup
+		errs xerrors.SyncErrors
+	)
+	for _, d := range append(s.Dependencies, s.Configuration) {
```

Comment:
Configuration should be stopped first ;)

Reply:
good catch!